.eslintrc.json

```diff
@@ -1,7 +1,17 @@
 {
+  "extends": [
+    "eslint:recommended",
+    "plugin:@typescript-eslint/recommended",
+    "plugin:react/recommended",
+    "plugin:jsx-a11y/recommended",
+    "plugin:react-hooks/recommended",
+    "plugin:jest/recommended",
+    "plugin:testing-library/react",
+    "plugin:react/jsx-runtime"
+  ],
```

Comment:
Removed `"plugin:storybook/recommended"` while migrating from the original `.eslintrc.json` since the package.json dependency was missing and causing the formatting to error out, can add back in if deemed necessary